### ISSUE TITLE
[Comment]: Section 9(a)(iii)(cb) Prior written approval ... to make a change to the business plan

### ISSUE THREAD

**Issue by NotableVentures** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#24]
### Comment

Requiring prior written approval for changes to a business plan -- Section 9(a)(iii)(cb) -- may unintentionally stifle innovation.  The functionality and business plan [sic] of a protocol are subject to constant iteration and improvements. Need language to allow for tinkering.

**Comment by Aaron-XReg** [comment]
Agreed! There should be a material change standard incorporated. CIMA does not need to be inundated by minor revision approval requests and this requirement will encourage firms to be much more selective in what they disclose to CIMA in their business plans.
